src/theme/card.css.ts

```diff
@@ -17,6 +17,7 @@ const base = style({
   padding: pxToRem(48),
   background: oryTheme.background.surface,
   color: oryTheme.foreground.def,
+  boxSizing: "border-box",
```

Comment:
Otherwise, we can't make the `wide` variant 100% (or would need to use `calc(100% - 96px)`). This seems neater, but requires the adjustment to the default width.

Reply:
i see, the PR seems to change quite a bit though. Since this affects the card used for the login / registration etc on Account Experience, could we just check with Figma and the Chromatic changes.

Reply:
You can see in Chromatic all the changes here https://www.chromatic.com/build?appId=63b58e306cfd32348fa48d50&number=183

Reply:
The figma document is here https://www.figma.com/file/XUGKyVy9UKakwbX7ialW7C/z-%2F-ory-account-experience?type=design&node-id=327-10217&mode=design&t=J2C8C1mymoklMtlp-0

Reply:
There aren't a lot of changes, it's just the border. I added it to the base-width, now there shouldn't be any changes.

Reply:
Figma says these should have a width of 432px, but on prod they are 434px (because of the missing `border-box` and the border of the cards).

Basically the whole web sets `box-sizing: border-box` for all elements, because it makes it easier to reason about... E.g. width: 50px means the element is 50px wide, no matter the border or padding applied.

Reply:
thank you! will take a look